src/lib/metrics.ts

```diff
@@ -327,6 +335,16 @@ export default class MetricsMonitor {
                         .observe(stage.duration);
                 });
 
+                stageCountByProject.reset();
+                stageCountByProjectResult.forEach((stageResult) =>
+                    stageCountByProject
+                        .labels({
+                            project_id: stageResult.project,
```

Comment:
we could also do it at the instance level but I Decided having per project info is more useful